src/org/matmaul/freeboxos/connection/xDslStatus.java

```diff
@@ -7,14 +7,39 @@
 @JsonAutoDetect(fieldVisibility = Visibility.ANY)
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class xDslStatus {
-	protected internalxDslStatus status;
+	protected internalXdslStatus status;
+	//protected internalXdslStats down;
+	//protected internalXdslStats up;
 	
-	public static class internalxDslStatus {
+	public static class internalXdslStatus {
 		public String status; 		// enum ro down, training,started,chan_analysis,msg_exchange,showtime,disabled
 		public String protocol; 	// enum ro t1414, adsl1_a, adsl2_a, adsl2plus_a, readsl2, adsl2_m, adsl2plus_m, unknown
 		public String modulation; 	// enum ro adsl, vdsl
 		public Long uptime;
 	}
+//	public static class internalXdslStats {
```

Comment:
This could be used in the future.
I tried to retrieve these data but then discovered that the API is returning other data fields that are not yet documented !
I think we can keep that  code commented and uncomment it later when the API is stable.